test/e2e/BUILD

```diff
@@ -46,10 +46,8 @@ go_library(
     name = "go_default_library",
     srcs = [
         "addon_update.go",
-        "apparmor.go",
         "audit.go",
```

Comment:
I think the audit test should be owned by sig-auth. I guess the alternative would be @kubernetes/sig-api-machinery-pr-reviews ? /cc @sttts